File: client/src/app/shared/Product.ts
export class Product {
    id: number;
    category: string;
    size: string;
    price: number;
    title: string;
    gameDescription: string;
    gameId: string;
    platform: string;
    gameReleaseDate: Date;

    constructor(id: number, category: string, size: string, price: number, title: string, gd: string,
        gid: string, p: string, grd: Date) {
        this.id = id;
        this.category = category;
        this.size = size;
        this.price = price;
        this.title = title;
        this.gameDescription = gd;
        this.gameId = gid;
        this.platform = p;
        this.gameReleaseDate = grd;
        
    }
}

File: Data/Entities/Product.cs
using System;
using System.Collections.Generic;
using System.ComponentModel.DataAnnotations;
using System.ComponentModel.DataAnnotations.Schema;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace GameDrewTotal.Data.Entities
{
  public class Product
  {
        public int Id { get; set; }
        public string Category { get; set; }
        public string Size { get; set; }
        public decimal Price { get; set; }
        public string Title { get; set; }
        public string GameDescription { get; set; }
        public string GameId { get; set; }
        public string Platform { get; set; }
        public DateTime GameReleaseDate { get; set; }
      }
}
